exchange/utils.go

```diff
@@ -741,3 +765,41 @@ func applyFPD(fpd *firstpartydata.ResolvedFirstPartyData, bidReq *openrtb2.BidRe
 		bidReq.User = fpd.User
 	}
 }
+
+func buildBidderRequestsWithBidResponses(req AuctionRequest, aliases map[string]string) map[openrtb_ext.BidderName]BidderRequest {
+	var bidderToBidderResponse map[openrtb_ext.BidderName]BidderRequest
+	if len(req.StoredBidResponses) > 0 {
+		// delete imps with stored bid resp
+		imps := req.BidRequest.Imp
+		req.BidRequest.Imp = nil //or new slice?
```

Comment:
Is this comment `//or new slice?` still needed?

Reply:
Removed, no need of this comment